pydsa/dijkstrashortestpath.py

```diff
@@ -0,0 +1,46 @@
+import heapq
+import sys
+
+
+
```

Comment:
Too many empty lines. Follow PEP8 standards.
You can use a plugin for checking pep8 in whatever code editor you use.